Sources/ParseSwift/Parse Types/GeoPoint.swift

```diff
@@ -1,15 +1,168 @@
 import Foundation
+import CoreLocation
```

Comment:
This will break compatibility with non-Apple platforms like Linux.

Reply:
Will remove as well